README.md

```diff
@@ -979,6 +979,7 @@ Table of Contents
   * [Plasmic](https://www.plasmic.app/) - A fast, easy to use, powerful web design tool and page builder that integrates into your codebase. Build responsive pages or complex components; optionally extend with code; and publish to production sites and apps.
   * [Proto.io](https://www.proto.io) - Create fully interactive UI prototypes without coding. Free tier available when free trial ends. Free tier includes: 1 user, 1 project, 5 prototypes, 100MB online storage and preview in proto.io app.
   * [resizeappicon.com](https://resizeappicon.com/) — A simple service to resize and manage your app icons.
+  * [Retool](https://retool.com) -  UI builder for admin panels and internal tools. Connects to any API and most databases. Free to build apps.
```

Comment:
"Free to build apps" perhaps doesnt describe this well, it's free only in editor mode meaning any user who have access to the app can edit it.  That's a bit weird so I'd rather pass on this one I think